lib/index.js

```diff
@@ -107,11 +109,38 @@ async function directoryExists(url) {
   ]).then(() => true).catch(() => false);
 }
 
+async function commandExists(command) {
+  return exec.exec(command, [], { silent: true }).then(() => true).catch(() => false);
+}
+
+async function installSlipstreamCLI() {
+  const commandExistsAlready = await commandExists('slipstream');
+
+  if (commandExistsAlready) {
+    return;
+  }
+
+  // install cli
+  core.startGroup('Installing the Slipstream CLI');
+  // for the future check that the binary exist
```

Comment:
Isn't this handled above now? Not sure what this comment means.

Reply:
It is handled, I will remove it 👍